charts/cinder-csi-plugin/templates/controllerplugin-statefulset.yaml

```diff
@@ -101,12 +101,12 @@ spec:
             - name: socket-dir
               mountPath: /csi
             - name: cacert
-              mountPath: /etc/kubernetes
+              mountPath: /etc/cacert
```

Comment:
Actually it is updated in docs , that it expects under /etc/kubernetes , https://github.com/kubernetes/cloud-provider-openstack/blob/master/docs/using-cinder-csi-plugin.md#using-the-helm-chart

Reply:
nope, doc said it's /etc/cacert
the cloud conf is /etc/kubernetes

Reply:
Ah!!